### PR TITLE
Add InvalidError for NativeAuth

### PR DESCRIPTION
**Pull request by SaurabhMSFT**
Corresponding PR in MSAL Repo : https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/2020

### CODE REVIEW

**Review by rmccahill — APPROVED** [review]

**Review by GBakolas — APPROVED** [review]

File: common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt [hunk 1]
```diff
@@ -72,4 +72,15 @@ interface INativeAuthCommandResult {
         open val correlationId: String = DiagnosticContext.INSTANCE.threadCorrelationId,
         open val errorCodes: List<Int>? = null
     )
+
+    data class InvalidUsername(
```
Comment (SammyO) [inline common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt:76]: Where is this class returned? I'm not seeing updates to the Controller or anywhere else.

File: common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt [hunk 2]
```diff
@@ -72,4 +72,15 @@ interface INativeAuthCommandResult {
         open val correlationId: String = DiagnosticContext.INSTANCE.threadCorrelationId,
         open val errorCodes: List<Int>? = null
     )
+
```
Comment (SammyO) [inline common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt:75]: If we're checking for blank usernames in the SDK interface, then we can remove the ` ArgUtils.validateNonNullArg(username, "username")` from the respective request classes right?

Comment (SaurabhMSFT) [reply common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt:75]: Done

File: common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt (hunk 1 above)
Comment (SaurabhMSFT) [reply common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt:76]: Updated

**Review by SammyO — APPROVED** [review]

**Pull request merged**